python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -214,6 +214,34 @@ def _checkProxy( delay=60, renew = True, shouldRaise = True, force = False ):
             last_modified_time = time.time()
 
 
+def getAccessURLs(lfns):
+    """
+    This is a function to get a list of the accessURLs
+    for a provided list of lfns.
+    """
+    # Get all the replicas
+    reps = execute('getReplicas(%s)' % lfns)
+    # Get the SEs
+    SEs = []
+    for lf in reps['Value']['Successful']:
+        for thisSE in reps['Value']['Successful'][lf].keys():
+            if thisSE not in SEs:
+                continue
```

Comment:
I'm guessing there was a mistake made here when editing as this set of loops does nothing.